module/grid/src/DataSet/DataSetGridId.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Grid\DataSet;
```

Comment:
This class use Grid andr DataSet but this is not any one of it. Sholud be in different namespace